### ISSUE TITLE
`continue` instead of `sharp left` or `left`

### ISSUE THREAD

**Issue by willwhite** [Project-OSRM/osrm-backend#4519]
Any idea why this maneuver is being treated as a `continue`? I would have expected it to be a `sharp left` or a `left`.

https://www.mapbox.com/get-directions/#19.76/37.36413/-121.91895?coordinates=-121.918555,37.364272;-121.918937,37.364137

**Comment by danpat** [comment]
@willwhite The maneuver is a `type: continue, modifier: left`.  I think the debate should be about whether this is a `continue` or a `turn`, but we at least did get the `left` bit basically correct :-)

**Comment by willwhite** [comment]
Good call. That's the thrust of my question. I understand `continue` to indicate that you're staying on the same road, which is not happening here.

**Comment by willwhite** [comment]
Running into the same problem here:

https://www.mapbox.com/get-directions/#20.56/37.74722/-122.40366?coordinates=-122.403659,37.74722;-122.40358837396278,37.747279926096155

**Comment by miccolis** [comment]
@willwhite this appears to be resolved. Can you confirm that it's working as expected?
